src/test/java/api/loans/scenarios/CheckInDeclaredLostItemTest.java

```diff
@@ -0,0 +1,477 @@
+      hasJsonPath("paymentStatus.name", "Transferred fully")));
+    verifyLostItemFeeAccountAction(
+      not(areTransferRefundActionsCreated(setCostFee)));
+
+    verifyLostItemProcessingFeeAccount(allOf(
+      hasJsonPath("amount", processingFee),
+      hasJsonPath("remaining", 0.0),
+      hasJsonPath("status.name", "Closed"),
+      hasJsonPath("paymentStatus.name", "Paid fully")));
+    verifyLostItemProcessingFeeAccountAction(
+      not(arePaymentRefundActionsCreated(processingFee)));
+  }
+
+  @Test
+  public void shouldRefundFeesWhenNoMaximumRefundPeriod() {
+    final double setCostFee = 10.55;
+    final double processingFee = 12.99;
+
+    useLostItemPolicy(lostItemFeePoliciesFixture.create(
+      lostItemFeePoliciesFixture.facultyStandardPolicy()
+        .withName("Test check in")
+        .chargeProcessingFee()
+        .withLostItemProcessingFee(processingFee)
+        .withSetCost(setCostFee)
+        .withNoFeeRefundInterval()).getId());
+
+    declareItemLost();
+
+    feeFineAccountFixture.transfer(feeFineAccountFixture
+      .getLostItemFeeAccount(loan.getId()), setCostFee);
+    feeFineAccountFixture.pay(feeFineAccountFixture
+      .getLostItemProcessingFeeAccount(loan.getId()), processingFee);
+
+    checkInFixture.checkInByBarcode(item);
+
+    verifyLostItemFeeAccount(isAccountRefundedFully(setCostFee));
+    verifyLostItemFeeAccountAction(areTransferRefundActionsCreated(setCostFee));
+
+    verifyLostItemProcessingFeeAccount(isAccountRefundedFully(processingFee));
+    verifyLostItemProcessingFeeAccountAction(arePaymentRefundActionsCreated(processingFee));
+  }
+
+  @Test
+  public void shouldCancelBothItemAndProcessingFeesWhenNeitherHaveBeenPaid() {
+    final double processingFee = 5.0;
+    final double itemFee = 10.0;
+
+    declareItemLost();
+
+    checkInFixture.checkInByBarcode(item);
+
+    verifyLostItemFeeAccount(isAccountClosed(itemFee));
+    verifyLostItemFeeAccountAction(isCloseActionCreated(itemFee));
+
+    verifyLostItemProcessingFeeAccount(isAccountClosed(processingFee));
+    verifyLostItemProcessingFeeAccountAction(isCloseActionCreated(processingFee));
+  }
+
+  @Test
+  public void canRefundTransferredFee() {
+    final double setCostFee = 10.89;
+    final double processingFee = 5.00;
+
+    useChargeableRefundableLostItemFee(setCostFee, processingFee);
+
+    declareItemLost();
+
+    final JsonObject lostItemFeeAccount = feeFineAccountFixture
+      .getLostItemFeeAccount(loan.getId());
+    feeFineAccountFixture.transfer(lostItemFeeAccount.getString("id"), setCostFee);
+
+    checkInFixture.checkInByBarcode(item);
+
+    verifyLostItemFeeAccount(isAccountRefundedFully(setCostFee));
+    verifyLostItemFeeAccountAction(areTransferRefundActionsCreated(setCostFee));
+
+    verifyLostItemProcessingFeeAccount(isAccountClosed(processingFee));
+    verifyLostItemProcessingFeeAccountAction(isCloseActionCreated(processingFee));
+  }
+
+  @Test
+  public void canRefundPaidFee() {
```

Comment:
Is this a fee that has been fully or partially paid? 

```suggestion
  public void shouldRefundPaidFee() {
```

Reply:
It is a fee that paid fully. There is `canCloseAndRefundPaidAndTransferredFee` test that covers partial transfer/payment